recipes/redlionfish/build.sh

```diff
@@ -0,0 +1 @@
+python ./setup_for_condabuild.py -vv install
```

Comment:
I don't understand why a special script is needed and it won't work with the normal `setup.py`? Everything [in there](https://github.com/rosalindfranklininstitute/RedLionfish/blob/main/setup.py) looks completely normal and compatible with conda-forge.

If so you can remove `build.sh` and `bld. bat` and add under `build:` below:
```yaml
build:
  script: "{{ PYTHON }} -m pip install . -vv"
```
which is our standard install line.